services/openshift/scripts/docker_config

```diff
@@ -1,28 +1,25 @@
 #!/usr/bin/env bash
 
-# Stop OpenShift Releated Containers
+#. /opt/adb/openshift/get_ip_address
```

Comment:
There is a typo `#` should not be present because it making this source include as comment.